### DISCUSSION TITLE
Tournament Client rounding SR to the closest tenth

### DISCUSSION THREAD

**Discussion by zachmanthethird** [ppy/osu#17005] [Q&A] [answered]
Version 2022.217.0 (tested locally in 2022.226.0 with same results)
https://osu.ppy.sh/b/1952065 4.75*
![image](https://user-images.githubusercontent.com/74262407/155871530-d8118518-9cd5-42cd-bb99-cdd02c36bd5d.png)
https://osu.ppy.sh/b/2596542 3.63* without DT, but OD/HP/SR didn't update based on mods
![image](https://user-images.githubusercontent.com/74262407/155871663-d3c350d8-9b00-4093-b04d-e8133d683c04.png)
More examples available here: https://www.twitch.tv/videos/1409546976

**Comment by Rekunan** [comment]
iirc, this is intentional so that stuff like 4* will show up as 4.0*
Though I do think it'd be better if it was 2 decimal places instead (so then my example would become 4.00*).

**Comment by peppy** [comment, marked as answer]
Will be addressed via https://github.com/ppy/osu/pull/17012
